llvm/include/llvm/IR/VPIntrinsics.def

```diff
@@ -587,6 +587,12 @@ VP_PROPERTY_FUNCTIONAL_OPC(Load)
 VP_PROPERTY_FUNCTIONAL_INTRINSIC(masked_load)
 END_REGISTER_VP(vp_load, VP_LOAD)
 
+BEGIN_REGISTER_VP_INTRINSIC(vp_load_ff, 1, 2)
+// val,chain = VP_LOAD_FF chain,base,mask,evl
```

Comment:
I was going to ask why this doesn't have an offset operand like the others so it could inherit from `VPBaseLoadStoreSDNode`, but then I realised we don't use the offset in RISCVISelLowering for regular vp.load/store.

SelectionDAGBuilder doesn't set it, I think it's always undef. Not for this PR but can we remove it?

Reply:
> I was going to ask why this doesn't have an offset operand like the others so it could inherit from `VPBaseLoadStoreSDNode`, but then I realised we don't use the offset in RISCVISelLowering for regular vp.load/store.
> 
> 
> 
> SelectionDAGBuilder doesn't set it, I think it's always undef. Not for this PR but can we remove it?

I think we can probably remove Offset and Addressing mode until a target comes along that needs them.